### ISSUE TITLE
Keybasefs doesn't let me login.

### ISSUE THREAD

**Issue by bnsh** [keybase/client#3402]
Hi. I'm trying to login to the keybasefs app... And, it's not letting me... I've attached screenshots. But, basically, it seems to be complaining about one of my keys that I made ages ago... I don't know what else beyond the screenshots to send, to help debug. (Although, of course, I'm not going to send my private keys...)

Any ideas?

Thanks!

![01-notlogged-in](https://cloud.githubusercontent.com/assets/353681/16719523/62796582-46f9-11e6-9ce0-54a90e76be36.png)
![02-clicked-login](https://cloud.githubusercontent.com/assets/353681/16719526/627d43aa-46f9-11e6-87a2-cddb38aa39af.png)
![03-enter-username](https://cloud.githubusercontent.com/assets/353681/16719524/627acf26-46f9-11e6-8482-cf721b35fd6a.png)
![04-enter-password](https://cloud.githubusercontent.com/assets/353681/16719525/627b055e-46f9-11e6-9a3b-7311957f74ab.png)
![05-sadness](https://cloud.githubusercontent.com/assets/353681/16719527/62801666-46f9-11e6-8c07-e194a4d0ce41.png)

**Comment by maxtaco** [comment]
Can you send the output of:

gpg --with-colons -k

And

gpg --with-colons -K

?

Thanks.

**Comment by bnsh** [comment]
Hi Max. Yep... (I just did this sed before I sent it (to preserve email privacy is all))

gpg --with-colons -k | sed -e 's/<[^<>:]*>/<email>/g' > gpgk.txt
gpg --with-colons -K | sed -e 's/<[^<>:]*>/<email>/g' > gpgK.txt

Hope that doesn't change anything.. There weren't any colons in any of the emails...

Thanks!

1. [gpgk.txt](https://github.com/keybase/client/files/357132/gpgk.txt)
2. [gpgK.txt](https://github.com/keybase/client/files/357133/gpgK.txt)

**Comment by bnsh** [comment]
Hmm... It's probably that there's extra stuff _after_ the <email> right? In that particular keys case, I had

"Options Analyzer <email> 2000-10-24"

And, it appears that in identity.go has a regex, that that particular key doesn't match. Perhaps I can simply change that key. But, I suppose, given that at least at one point I was able to make that key, maybe the regex should be changed?

**Comment by bnsh** [comment]
Just a python script that verifies that that key is in fact the problem... On my box, it outputs

```
pub	:::Options Analyzer <automatic@cablemodem.hex21.com> 2000-10-24:::	None
```

So, I think it's the date afterwards that's causing it's woes..

[problem.zip](https://github.com/keybase/client/files/357418/problem.zip)

**Comment by bnsh** [comment]
Yep. After I deleted that uid, and recreated another one in the "proper" way: "Options Analyzer (2000-10-24) <automatic@cablemodem.hex21.com>"

I'm able to log in again. If you want, you can close this... But, perhaps other people may have made keys in the ancient past that they might have to do the same with...
